.github/workflows/ci.yml

```diff
@@ -5,52 +5,39 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
     - uses: actions/checkout@v1
-    - name: Install wget and configure Microsoft apt repository
+    - name: Fetch Secp256k1.Native
```

Comment:
I'd prefer to build and pack BC-DNL.Core before fetching any dependencies.

Reply:
But then we'd have to reinstall NetCore 3 because we have removed it. What is the advantage of building BouncyCastle first? I am running `dotnet clean` and removing the native dependency.

Reply:
Reinstalling .NETCore3 is fine. I'm just paranoid of bringing a binary blob (the one from myget) before building our nuget package.